src/analysis/normed_space/hahn_banach.lean

```diff
@@ -95,48 +124,57 @@ begin
   { exact f.op_norm_le_bound g.extend_to_ℂ.op_norm_nonneg (λ x, h x ▸ g.extend_to_ℂ.le_op_norm x) },
 end
 
+instance complex_has_exists_extension_norm_eq : has_exists_extension_norm_eq ℂ :=
+⟨by { intros, apply complex.exists_extension_norm_eq }⟩
+
 end complex
 
 section dual_vector
-variables {E : Type*} [normed_group E] [normed_space ℝ E]
+variables {𝕜 : Type v} [nondiscrete_normed_field 𝕜] [normed_algebra ℝ 𝕜]
+variables {E : Type u} [normed_group E] [normed_space 𝕜 E]
 
 open continuous_linear_equiv
 open_locale classical
 
-lemma coord_self' (x : E) (h : x ≠ 0) : (∥x∥ • (coord ℝ x h))
-  ⟨x, submodule.mem_span_singleton_self x⟩ = ∥x∥ :=
-calc (∥x∥ • (coord ℝ x h)) ⟨x, submodule.mem_span_singleton_self x⟩
-    = ∥x∥ • (linear_equiv.coord ℝ E x h) ⟨x, submodule.mem_span_singleton_self x⟩ : rfl
-... = ∥x∥ • 1 : by rw linear_equiv.coord_self ℝ E x h
-... = ∥x∥ : mul_one _
+lemma coord_self' (x : E) (h : x ≠ 0) :
```

Comment:
The name of the lemma is not very evocative. Could you try to come up with a better name, or at least include a docstring because the lemma is not very easy to parse?

Reply:
I don't really know what this is about; @hrmacbeth ?